tests/labs/lie_functional/common.py

```diff
@@ -6,6 +6,9 @@
 import torch
 
 
+TEST_EPS = 5e-7
```

Comment:
Where was this before getting added here? I didn't see another file in this PR using it.

Reply:
Before it was imported from here, meaning that labs code was peaking into Theseus main code. 

<img width="655" alt="image" src="https://user-images.githubusercontent.com/4759586/213213446-dea86590-79f9-473f-8172-30589c48eafa.png">

Reply:
Ah!